ctru-rs/src/services/hid.rs

```diff
@@ -282,11 +317,195 @@ impl Hid {
 
         (res.dx, res.dy)
     }
+
+    /// Returns the current volume slider position (between 0 and 1).
+    ///
+    /// # Notes
+    ///
+    /// The [`ndsp`](crate::services::ndsp) service automatically uses the volume slider's position to handle audio mixing.
+    /// As such this method should not be used to programmatically change the volume.
+    ///
+    /// Its purpose is only to inform the program of the volume slider's position (e.g. checking if the user has muted the audio).
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// # use std::error::Error;
+    /// # fn main() -> Result<(), Box<dyn Error>> {
+    /// #
+    /// use ctru::services::hid::Hid;
+    /// let mut hid = Hid::new()?;
+    ///
+    /// hid.scan_input();
+    ///
+    /// let volume = hid.volume_slider();
+    /// #
+    /// # Ok(())
+    /// # }
+    /// ```
+    #[doc(alias = "HIDUSER_GetSoundVolume")]
+    pub fn volume_slider(&self) -> f32 {
```

Comment:
I feel like it should be clear that this is not the actual volume slider reading but the volume reading.

For the volume slider, 3D slider, etc there's `mcuhwc` to wrap still

Reply:
I believe it is the actual volume slider reading. Where did you get this information?

Reply:
well... the MCU is the device handling these peripherals, and `mcu:hwc` communicates directly with it.

If you read from the MCU and move the slider, the data it reads is pretty "coarse".

https://github.com/devkitPro/libctru/blob/c579a65655900f6fd1bcbad42b0d68d80caba303/libctru/source/services/mcuhwc.c#L117C1-L129C1

Reply:
That’s true, but it also wouldn’t be the only time that HorizonOS leaves multiple ways to handle the same task. 😅

It’s possible they are equally as correct, or maybe one is a more sophisticated (and possibly differently working) version of the other. In my opinion it makes sense for the slider to be in HID, together with the other faceplate buttons.